### ISSUE TITLE
Crash in Reader / ReaderStreamViewController 

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-iOS#9997]
New in 10.6: 

```
#0. Crashed: com.apple.main-thread
0  WordPress                      0x1052d7984 ReaderStreamViewController.displayResultsStatus() (ReaderStreamViewController.swift:467)
1  WordPress                      0x1052e16f0 specialized closure #2 in ReaderStreamViewController.fetchTagTopic() (ReaderStreamViewController.swift:407)
2  WordPress                      0x1052e7bb8 partial apply for closure #2 in ReaderStreamViewController.fetchSiteTopic() (ReaderStreamViewController.swift)
3  WordPress                      0x105452b34 _T0s5Error_pSgIegx_So7NSErrorCSgIeyBy_TR (ThemeBrowserViewController.swift)
4  WordPress                      0x10502a2c4 __68-[ReaderTopicService siteTopicForSiteWithID:isFeed:success:failure:]_block_invoke.405 (ReaderTopicService.m:624)
5  WordPressKit                   0x1066276dc __78-[ReaderTopicServiceRemote fetchSiteInfoForSiteWithID:isFeed:success:failure:]_block_invoke.179 (ReaderTopicServiceRemote.m:234)
6  WordPressKit                   0x1066ff58c _T0So7NSErrorCSo15HTTPURLResponseCSgIeyByy_AbEIegxx_TRTA (WordPressComRestApi.swift)
7  WordPressKit                   0x1066fccb0 closure #1 in closure #3 in WordPressComRestApi.multipartPOST(_:parameters:fileParts:requestEnqueued:success:failure:) (WordPressComRestApi.swift:276)
8  Alamofire                      0x105bde168 partial apply for specialized  (ResponseSerialization.swift:167)
9  Alamofire                      0x105bba4b0 _T0Ieg_IeyB_TR (NetworkReachabilityManager.swift)
10 libdispatch.dylib              0x183b9caa0 _dispatch_call_block_and_release + 24
11 libdispatch.dylib              0x183b9ca60 _dispatch_client_callout + 16
12 libdispatch.dylib              0x183bddd80 _dispatch_main_queue_callback_4CF$VARIANT$armv81 + 964
13 CoreFoundation                 0x184253070 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 12
14 CoreFoundation                 0x184250bc8 __CFRunLoopRun + 2272
15 CoreFoundation                 0x184170da8 CFRunLoopRunSpecific + 552
16 GraphicsServices               0x186156020 GSEventRunModal + 100
17 UIKit                          0x18e190758 UIApplicationMain + 236
18 WordPress                      0x104f505c0 main (main.m:11)
19 libdyld.dylib                  0x183c01fc0 start + 4
``` 

5b7185f66007d59fcdaa7477-fabric

**Comment by loremattei** [comment]
We still have it in 10.9: 5baabbddf8b88c29634d7eb1-fabric

**Comment by etoledom** [comment]
Fixed (hopefully) on #10235
